packages/cli/test/utils/simulation/beacon_clients/lighthouse.ts

```diff
@@ -92,10 +91,10 @@ export const generateLighthouseBeaconNode: BeaconNodeGenerator<BeaconClient.Ligh
       },
       health: async () => {
         try {
-          await got.get(`http://127.0.0.1:${ports.beacon.httpPort}/eth/v1/node/health`);
+          await fetch(`http://127.0.0.1:${ports.beacon.httpPort}/eth/v1/node/health`);
```

Comment:
please make sure to always use wrapped fetch function to improve the error handling

https://github.com/ChainSafe/lodestar/blob/6675739aba0aacd904109bf06faaf74a7b41ec6d/packages/api/src/utils/client/fetch.ts#L14

Reply:
put up a PR to enforce this via eslint rules
- https://github.com/ChainSafe/lodestar/pull/6500

Reply:
The lint rule has been merged into our unstable branch, please make sure to update your branch so that we correctly update all fetch references and use wrapped fetch instead of directly calling native fetch